pkg/providers/securitygroup/suite_test.go

```diff
@@ -72,9 +73,9 @@ var _ = BeforeSuite(func() {
 	}
 	ctx = settingsStore.InjectSettings(ctx)
 	ctx, stop = context.WithCancel(ctx)
-
+	TTL := 5 * time.Minute
```

Comment:
You don't need to define the TTL value here, you can just use the exported variable in the securitygroup package. i.e. `securitygroup.TTL`